src/NftReward.sol

```diff
@@ -49,22 +50,25 @@ contract NftReward is ERC721, Ownable, Pausable, EIP712 {
     }
 
```

Comment:
Shouldn't we also disable initializers (https://docs.openzeppelin.com/upgrades-plugins/1.x/writing-upgradeable#initializing_the_implementation_contract)?

Reply:
Thank you :)
Found [this article](https://forum.openzeppelin.com/t/uupsupgradeable-vulnerability-post-mortem/15680) that help ed me.